### PR TITLE
Treasure die

### LINKED ISSUE #141: Missing results of treasure die
image

### PR DESCRIPTION
**Pull request by DarthGandalf**
Fix #141

### CODE REVIEW

**Comment by qwrtln** [comment]
Please post before/after pics.

**Comment by DarthGandalf** [comment]
before
after

File: sections/all_map_locations.tex [hunk 1]
```diff
@@ -8,11 +8,12 @@ \subsection*{Symbols on the Map}
 \begingroup
   \renewcommand{\arraystretch}{1.5}
   \newcolumntype{Y}{>{\centering\arraybackslash}X}
-  \begin{tabularx}{\linewidth}{Y m{0.7\linewidth}}
+  \begin{tabularx}{\linewidth}{Y m{0.8\linewidth}}
     \small
     \mbox{\textbf{I-VII}} & Field Difficulty corresponding to the level of Neutral Units. \\
-    \includesvg[height=15px]{\svgs/treasure.svg} & Roll a Treasure Die and gain the indicated bonus. \\
+	\includesvg[height=15px]{\svgs/treasure.svg} & Roll a Treasure Die and gain the indicated bonus (\includesvg[height=15px]{\svgs/resource_die.svg}, \includesvg[height=15px]{\svgs/2_treasure_die.svg}, \includesvg[height=15px]{\svgs/experience.svg}, or \includesvg[height=15px]{\svgs/artifact.svg}).\index{Treasure Die} \\
```
Comment (qwrtln) [inline sections/all_map_locations.tex:14]: As they're inline, not serving as bullet points, maybe make them smaller? Say 10px?

File: sections/all_map_locations.tex [hunk 2]
```diff
@@ -8,11 +8,12 @@ \subsection*{Symbols on the Map}
 \begingroup
   \renewcommand{\arraystretch}{1.5}
   \newcolumntype{Y}{>{\centering\arraybackslash}X}
-  \begin{tabularx}{\linewidth}{Y m{0.7\linewidth}}
+  \begin{tabularx}{\linewidth}{Y m{0.8\linewidth}}
     \small
     \mbox{\textbf{I-VII}} & Field Difficulty corresponding to the level of Neutral Units. \\
-    \includesvg[height=15px]{\svgs/treasure.svg} & Roll a Treasure Die and gain the indicated bonus. \\
+	\includesvg[height=15px]{\svgs/treasure.svg} & Roll a Treasure Die and gain the indicated bonus (\includesvg[height=10px]{\svgs/resource_die.svg}, \includesvg[height=10px]{\svgs/2_treasure_die.svg}, \includesvg[height=10px]{\svgs/experience.svg}, or \includesvg[height=10px]{\svgs/artifact.svg}).\index{Treasure Die} \\
```
Comment (qwrtln) [inline sections/all_map_locations.tex:14]:
```suggestion
    \includesvg[height=15px]{\svgs/treasure.svg} & Roll a Treasure Die and gain the indicated bonus (\includesvg[height=10px]{\svgs/resource_die.svg}, \includesvg[height=10px]{\svgs/2_treasure_die.svg}, \includesvg[height=10px]{\svgs/experience.svg}, or \includesvg[height=10px]{\svgs/artifact.svg}).\index{Treasure Die} \\
```

Comment (qwrtln) [reply sections/all_map_locations.tex:14]: Huh, the linter should have picked it up.

Comment (qwrtln) [reply sections/all_map_locations.tex:14]: Argh, it did, but it didn't fail xD
https://github.com/Heegu-sama/Homm3BG/actions/runs/8734963836/job/23966726435?pr=183

**Comment by qwrtln** [comment]
I'm sorry, but can you please indulge my OCD? Please swap Spell and Arficat rows in the table, so that the ordering follows Treasure Die results.

image

**Comment by qwrtln** [comment]
Oh, one last thing... `po4a` 😆 This section hasn't been translated yet, so no harm.

**Review by Heegu-sama — APPROVED** [review]

**Pull request merged**